airflow/utils/log/file_task_handler.py

```diff
@@ -179,6 +179,8 @@ class FileTaskHandler(logging.Handler):
     inherits_from_empty_operator_log_message = (
         "Operator inherits from empty operator and thus does not have logs"
     )
+    executor_instances: dict[str, BaseExecutor] = {}
+    default_executor_key = "_default_executor"
```

Comment:
Looks like a constant to me. we probably could capitalize it
```suggestion
    DEFAULT_EXECUTOR_KEY = "_default_executor"
```